teams/infra/configure-maintenance-pull-request/action.yml

```diff
@@ -0,0 +1,81 @@
+name: Configure a maintenance Pull Request
+
+description: Configure a maintenance Pull Request according to the Infra team guidelines
+
+inputs:
+  github-token:
+    description: A GitHub access token with write access to the project
+    required: true
+    type: string
+
+runs:
+  using: composite
+  steps:
+  # Set global values as outputs to be reused in other steps
+  - id: global # Maintenance parameters of the Infra team
+    env:
+      output-maintenance-label: dependency-upgrade
```

Comment:
I applied your suggestion and pushed my rework.

Reply:
I find the use of GH expressions nice (when one keep things simple) because it allows to keep the configuration as declarative as possible (avoid bash scripts). Just with the name of the variable/output you can get a first understanding of what you will get without having to read the expression.

I was planning to create a very basic action to set outputs from input. Unfortunately composite action does not support dynamic outputs (unlike docker or javascript action).
```yaml
name: example
on:
  pull_request:
jobs:
  configure-pr:
    runs-on: ubuntu-latest
    steps:
    - id: global
      uses: camunda/infra-global-github-actions/set-outputs@main
      with:
        first-output-to-set: ${{ expression/value }}
        second-output-to-set: ${{ expression/value }}
    - run: |
        echo ${{ steps.global.outputs.first-output-to-set:}
```      

I found this interesting [action](https://github.com/marketplace/actions/object-remapping-from-json-to-json) on the marketplace but it is not what I am looking for.

It may be worthwhile to develop this action in Javascript (for dynamic outputs) if we want to use more this style in the future.